src/environment.js

```diff
@@ -135,6 +135,13 @@ export default class Environment extends EventEmitter {
       this.dir = process.cwd();
     }
 
+    if (!this.dest) {
+      this.dest = 'sassdoc';
+    }
+
+    this.dest = this.resolve(this.dest, this.destCli);
+    this.destName = path.relative(process.cwd(), this.dest);
```

Comment:
You're right, though `themeName` can also be the package name (not path)… and it's used only for display.

We could call them `destPath` and `themePath` but it would not be relevant for `theme` in the case of a non-path package name.

I'm thinking the most correct way would be to rename them in `displayDest` and `displayTheme`.

But `themeName` was already present in the interface, I don't think it's documented though, so we can consider it private and might rename it without "breaking".